wiki/Shortcut_key_reference/nl.md

```diff
@@ -0,0 +1,290 @@
+---
+tags:
+  - hotkey
+  - hotkeys
+  - keyboard
+  - shortcuts
+---
+
+# Sneltoets combinaties
+
+Osu! heeft een groot aantal sneltoets combinaties die kunnnen worden gebruikt in-game of wanneer je op een bepaalt scherm bent. De combinaties en hun functies zijn hieronder vermeld.
+
+*Opmerking: Dit artikel geeft de standaard sneltoets combinaties weer. Je kunt enkele hiervan veranderen in de [opties](/wiki/Options).*
+
+## Game-wide
+
+Deze combinaties werken overal:
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Alt` + `F4` | Sluit de game client af (zonder waarschuwing). Als je een beatmap aan het spelen bent of in de beatmap editor zit keer je terug naar het liedjes-selectie scherm. Als je in osu!direct zit, keer je terug naar het hoofdmenu. |
+| `Alt` + `Up/Down Arrow` | Veranderd het hoofdvolume. Gebruik de linker- en rechter pijlen om te switchen naar effect en muziek volumes. |
+| `Ctrl` + `Alt` + `Shift` + `S` | Herlaad de huidige skin. Als je deze combinatie indrukt tijdens gameplay, zal dit gebeuren nadat je terugkeert naar het liedjes-selectie scherm. |
+| `Ctrl` + `F11` | Toont/verbergt de frame times graph. Druk op `Ctrl` om deze te pauzeren. |
+| `F7` | Cirkel door de frame rate limiters. |
+| `F8` | Toont/verbergt de chat window |
+| `F9` | Toont/verbergt de uitgebreide chat window. |
+| `F10` | Zet alle muisknoppen aan/uit (behalve het muiswiel, deze is altijd aan). |
+| `F12` | Neemt een screenshot. Klik op de dialoog om de screenshots folder te openen. |
+| `Shift` + `F12` | Neemt een screenshot en upload deze naar de server. De screenshot wordt ook lokaal opgeslagen. De screenshot wordt ook geopent in je web browser. |
+| `Insert` | Dit is de [boss key](https://en.wikipedia.org/wiki/Boss_key "Boss key op Wikipedia"). Verkleint de game client naar het notificatie vak. De game wordt ook gedempt. |
+| `Shift` + `F1` | Opent [osu!status](https://twitter.com/osustatus "osu!status op Twitter") in je web browser. (Alleen als er een connectie error verschijnt.) |
+| `Ctrl` + `O` | Opent de opties sidebalk. Je kunt de offset wizard niet gebruiken als je niet in het hoofdmenu bent. |
+
+## Hoofdmenu
+
+| Sneltoets | Actie |
+| :-- | :-- |
+| `Alt` + `Enter` | Zet de volledige scherm modus aan/uit. |
+| `Up/Down Arrow` | Veranderd het hoofdvolume. Gebruik de linker- en rechter pijlen om te switchen naar effect en muziek volumes. |
```

Comment:
| `Up/Down Arrow` | Verandert het hoofdvolume. Gebruik de linker- en rechter pijltjestoetsen om te schakelen naar effect- of muziekvolume.